src/widgets/GitResetToRemoteForm.tsx

```diff
@@ -0,0 +1,68 @@
+import { Dialog } from '@jupyterlab/apputils';
+import { TranslationBundle } from '@jupyterlab/translation';
+import { Widget } from '@lumino/widgets';
+import { Git } from '../tokens';
+
+/**
+ * The UI used for the Reset to Remote Form,
+ * used as the body for Reset to Remote Dialog.
+ */
+export class GitResetToRemoteForm
```

Comment:
Very nice refactoring. But let's make the naming generic so we could use this dialog with checkbox in other places.

```suggestion
export class CheckboxForm
```